docs/self-managed/components/orchestration-cluster/core-settings/concepts/data-retention.md

```diff
@@ -0,0 +1,26 @@
+---
+id: data-retention
+title: Data retention
+description: "Overview of how the Orchestration Cluster stores and archives data in Elasticsearch or OpenSearch."
+---
+
+The Orchestration Cluster centrally manages data retention for all data using unified storage and policy configuration.
+
+All cluster data, including deployed process definitions, process instance state, user operations, and technical metadata, is written to secondary storage in Elasticsearch or OpenSearch. The data representing process instance state becomes immutable after the process instance is finished, and it becomes eligible for archiving.
+
+During data archive, data is moved to a dated index (e.g., `operate-variable_2020-01-01`), with the suffix representing the completion date of the associated process or operation.
+
+:::note
+All data present in the Database (from both **main** and **dated** indices) is visible from the UI.
+:::
+
+## Archive period
+
+By default, the time between a process instance finishing and being moved to a dated index is one hour. This can be modified using the [waitPeriodBeforeArchiving](/self-managed/components/orchestration-cluster/zeebe/exporters/camunda-exporter.md#configurations) configuration parameter.
```

Comment:
maybe we shouldn't give the actual default here, but rather point to the configuration where the default is listed. If we ever change that default, it is unlikely that we would remember to update this line too